<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contact_editor</class>
 <widget class="QDialog" name="Contact_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>170</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="name_value"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="uri_label">
        <property name="text">
         <string>URI</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="uri_value"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="accept_btn">
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="dismiss_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dismiss</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
